### PR TITLE
Update package.py for totalview for latest release 2023.1.6

### PR DESCRIPTION
**Pull request by elliebinkley**
Update package.py for totalview for latest release 2023.1.6
Fix URLS for previous releases that were causing a failure on spack install. 
Update maintainers to include Larry Burley ( elliebinkley)

### CODE REVIEW

**Review by petertea — APPROVED** [review]
I've worked with Larry on this and understanding why my previous commit did not seem to work in a new environment.  We found that the URL did not work with a relative name, so adding back the format(os.getcwd()) was in fact necessary.  The changes look good to me.

**Review by dshrader — APPROVED** [review]

File: var/spack/repos/builtin/packages/totalview/package.py [hunk 1]
```diff
@@ -2,6 +2,11 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+# Update Log
+#
+# Apr 13 2023 LB:Update for totalview@2023.1.6 version
+#
+#
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/totalview/package.py:9]:
```suggestion
```

File: var/spack/repos/builtin/packages/totalview/package.py [hunk 2]
```diff
@@ -20,7 +25,7 @@ class Totalview(Package):
     be downloaded as a resource."""
 
     homepage = "https://totalview.io"
-    maintainers("dshrader", "petertea")
+    maintainers = ["dshrader", "petertea", "elliebinkley"]
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/totalview/package.py:28]: Please keep the directive, instead of using a list

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**